www/jana19/cv08/buy.php

```diff
@@ -0,0 +1,17 @@
+<?php
+session_start();
+require_once __DIR__ . '/db/ProductsDB.php';
+# session pole pro kosik
+if (!isset($_SESSION['cart'])) {
+    $_SESSION['cart'] = [
+        
+    ];
+}
+
+$productsDB = new ProductDB();
+$goodsFound = $productsDB->findItemsByID();

```

Comment:
muzete se podivat na to co se vrati z metody `findItemsByID()`?

Reply:
Děkuji, povedlo e mi to opravit.
Ostatní jsem bohužel nestihla. Měla jsem to v plánu na čtvrtek, ale netušila jsem, že je v ten den "sportovní den" a já to pódium měla přímo pod oknem, omlouvám se, ale to opravdu nešlo v tom hluku.
Musím teď prioritně řešit BP, ale později se na toto a další úkol podívám. (S tím, že nebudu moct dostat body počítám, rozumím, chápu.)